libopenage/CMakeLists.txt

```diff
@@ -87,8 +87,14 @@ find_package(Epoxy REQUIRED)
 find_package(HarfBuzz 1.0.0 REQUIRED)
 
 set(QT_VERSION_REQ "5.4")
-find_package(Qt5Core ${QT_VERSION_REQ} REQUIRED)
-find_package(Qt5Quick ${QT_VERSION_REQ} REQUIRED)
+if(APPLE)
+	# tell CMake to look in the default homebrew installation for Qt5
+	find_package(Qt5Core ${QT_VERSION_REQ} REQUIRED PATHS "/usr/local/Cellar/qt5/*")
+	find_package(Qt5Quick ${QT_VERSION_REQ} REQUIRED PATHS "/usr/local/Cellar/qt5/*")
```

Comment:
I also think that the real problem is that cmake is not able to find cmake definitions for Qt5, so setting `CMAKE_PREFIX_PATH` is the correct solution (it is just one step, instead of having to specify path to qt on each invocation to `find_package` e.g. as new packages are desired in the future). Since Qt is installed by homebrew to non-standard location and the install warns that it is not linked directly into `/usr/local`, then either the user should specify the CMAKE_PREFIX_PATH as an environment variable themselves (it in principle is the user's responsibility to inform the build system of non-standard build locations), or we can do it ourselves to make things easier for 95% of them:

```
if(APPLE)
        list(APPEND CMAKE_PREFIX_PATH /usr/local/opt/qt5)
endif()
```

In both cases, there will not be a need for an `if(APPLE)` for `find_package` invocations here then.